.nvmrc

```diff
@@ -0,0 +1 @@
+v16.10.0
```

Comment:
Newline + `16.6.1` is what template & eslint-config run. If we wanna go to latest minor, we should update template, or we should use template here

Reply:
v16.6.1 wasn't supported with some of the other dependencies, so I've bumped it. Added white space. 
@andersonaguiar  or @xenide may want to look at whether that can be resolved - I didn't dig into the details much, just followed the advice in the npm install output.